handout/directives/attribute_directives.md

```diff
@@ -1,5 +1,5 @@
 # Attribute Directives
 
-Attribute directives are a way of changing the appearance or behavior of a component. Ideally, a directive should work in a way that is component agnostic and not bound to implementation details.
+Attribute directives are a way of changing the appearance or behavior of a DOM element. Ideally, a directive should work in a way that is component agnostic and not bound to implementation details.
```

Comment:
From the Angular site, it uses DOM elements, "An Attribute directive changes the appearance or behavior of a DOM element."

This change makes the correction.